cwd/app.js

```diff
@@ -46,27 +48,28 @@
         }
 
         if(SHOW_ONE_TITLE) {
-          var arr = Array.prototype.slice.call(document.getElementsByClassName("glow-on-hover"));
-          arr = arr.slice(0,3);
-          var index = arr.indexOf(document.getElementById(this.id));
+          let buttons = Array.prototype.slice.call(document.getElementsByClassName("glow-on-hover"));
+          buttons = buttons.filter(obj => obj.id.includes("Button"));
 
-          arr[index].style.display = "none";
+          const index = buttons.indexOf(document.getElementById(this.id));
```

Comment:
Duplicate naming of a global constant `index`.  I feel like the global constant is important enough not to be worth overriding it.

Reply:
Yeah, the one is global because I moved it in response to a suggestion you made earlier. I wonder if it would be better to put it back where it originally was so there is no global index. Also this issue might be fixed by the linking of the three elements into one object with a shared index like you suggested.

Reply:
`const` is only used for a new declaration, no?  Just a little confused by if this is truly the same variable `index` or not.

Reply:
We discussed this a bit in person, this has been fixed by moving the other index variable back to where it belongs, not as a global variable.